backend/test/services/model.spec.ts

```diff
@@ -418,7 +418,7 @@ describe('services > model', () => {
   test('setLatestImportedModelCard > success', async () => {
     await setLatestImportedModelCard('abc')
 
-    expect(modelMocks.updateOne).toHaveBeenCalledOnce
+    expect(modelMocks.updateOne).toHaveBeenCalledOnce()
```

Comment:
I have a feeling this unit test was already broken, but now that we're actually calling `toHaveBeenCalledOnce()` it's correctly showing that it's failed.